shuttle/templates/driver_check_in/load_driver_check_locations.html

```diff
@@ -1,47 +1,52 @@
 {% block body_content %}
     <!-- TODO: grabbing locations from sheets takes some time to load. Is there a way to load this info faster? -->
+    <div class="center-page">
+        <h4>
+            You are scheduled for <span class="badge badge-pill badge-warning location-status">{{ next_location }}</span>
```

Comment:
Could you change this to be a little more descriptive. Maybe something like

"You are scheduled to arrive/depart at/from <location> at <time>

Depending on which it is showing

Reply:
Well rn it's just showing the next stop so what about "Your next stop is scheduled for ___ at ___?" or "You are scheduled to arrive at". Or do you want it to show both arrival and depart at all times?

Reply:
I believe that all the times in the schedule are departure times and that is where I assume you are getting the time from so probably the depart option

Reply:
Ah gotcha